docs/UserGuide.md

```diff
@@ -443,7 +443,21 @@ Action | Format, Examples
 
 <div style="text-align: right"><a href="https://ay2021s1-cs2103t-f13-4.github.io/tp/UserGuide.html">^ Back to top</a></div>
 
-# 7. Glossary
+## 7. Glossary
```

Comment:
Will it be better for the terms in the table to be sorted by alphabetical order?

Reply:
Okay changed, thanks.